docs/anomaly-detection/components/models/models.md

```diff
@@ -267,12 +255,11 @@ Here we use Isolation Forest implementation from `scikit-learn` [library](https:
 
 *Parameters specific for vmanomaly*:
 
-\* - mandatory parameters.
-* `class`\* (string) - model class name `"model.isolation_forest.IsolationForestMultivariateModel"`
+* `class` (string) - model class name `"model.isolation_forest.IsolationForestMultivariateModel"`
 
-* `contamination` - The amount of contamination of the data set, i.e. the proportion of outliers in the data set. Used when fitting to define the threshold on the scores of the samples. Default value - "auto". Should be either `"auto"` or be in the range (0.0, 0.5].
+* `contamination` (float|string, optional) - The amount of contamination of the data set, i.e. the proportion of outliers in the data set. Used when fitting to define the threshold on the scores of the samples. Default value - "auto". Should be either `"auto"` or be in the range (0.0, 0.5].
```

Comment:
markdown is broken here, assume its due to un-escaped `|` that is tried to be rendered as table row/column
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/11490990/14699f0a-cf73-4220-96d4-1a3475c2782b)